docs/src/app/(public)/(content)/layout.tsx

```diff
@@ -25,7 +25,7 @@ export default function Layout({ children }: React.PropsWithChildren) {
         ))}
       </SideNav.Root>
 
-      <main className="ContentLayoutMain">
+      <main className="ContentLayoutMain" id="main-content">
```

Comment:
Nice, to see it uses the same id name as in Material UI. A different name would be a pain to manage in the Algolia config. More context: https://github.com/algolia/docsearch/issues/1745.